### PR TITLE
#168675509 Setup postgres 

### PR DESCRIPTION
**Pull request by oleesir**
#### What does this PR do?
This PR implements the endpoint setup Postgres
#### Description of Task to be completed?
Setup Postgres database
#### How should this be manually tested?
- clone the git repo and run 'npm install' to download the necessary dependencies.
- In the command line type, 'npm run migrate:db' to create the db in the developement environment.
- In the command line type, 'npm run migrate:test_db' to create the db in the test environment.
#### What are the relevant pivotal tracker stories?
[#168675509](https://www.pivotaltracker.com/story/show/168675509)
#### Any background context you want to add?
N/A
#### Screenshots
N/A

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,6 +3,15 @@ jobs:
   build:
     docker:
       - image: circleci/node:10.16.3
+        environment:
```
Comment (otseobande) [inline .circleci/config.yml:7]: I don't think you need this block under the node image

File: server/src/database/migrations/20200422063820-create-users.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true
+    },
+    password: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    type: {
```
Comment (otseobande) [inline server/src/database/migrations/20200422063820-create-users.js]: Let's rename this field to "role"

**Review by otseobande — COMMENTED** [review]
Looks good. I think you worked well on this. Please attend to the Hound comments also.

File: server/src/database/migrations/20200422063820-create-users.js (hunk 2 above)
Comment (oleesir) [reply server/src/database/migrations/20200422063820-create-users.js]: done

**Pull request merged**